include/triton/core/tritonserver.h

```diff
@@ -1634,6 +1634,18 @@ TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_ServerLoadModel(
 TRITONSERVER_DECLSPEC TRITONSERVER_Error* TRITONSERVER_ServerUnloadModel(
     TRITONSERVER_Server* server, const char* model_name);
 
+/// Unload the requested model, and also unload the models that only be used
```

Comment:
"Unload the requested model, and also unload any dependent model that was loaded along with the requested model (for example, the models composing an ensemble). Unloading a model ... will be returned. The function does not return until the requested model and all dependent models are unloaded or fail to unload. Returned error indicates if all models unloaded successfully or not."